packages/teams/src/utils/get-user-creatable-teams.ts

```diff
@@ -12,12 +13,17 @@ import { IGroupTemplate } from "../types";
  */
 export function getUserCreatableTeams(
   user: IUser,
-  environment: HubProduct
+  environment: HubProduct,
+  currentVersion: string
 ): IGroupTemplate[] {
+  // TODO: remove this when needed
```

Comment:
Put a date in here so next dev can know when they can remove it... i.e. after May 1 2021

Reply:
Could we delegate to `getTeamsAvailableInProduct(...)` in this fn so we don't have the current version parse/compare logic twice?